packages/now-node/src/index.ts

```diff
@@ -429,7 +429,7 @@ export async function startDevServer({
   });
   const { pid } = child;
   const onMessage = once<{ port: number }>(child, 'message');
-  const onExit = once<{ code: number; signal: string | null }>(child, 'exit');
+  const onExit = once.spread<[number, string | null]>(child, 'exit');
```

Comment:
Is `once.spread()` documented somewhere?

Reply:
https://github.com/TooTallNate/once#oncespreadtemitter-eventemitter-name-string-cancelablepromiset